angular.json

```diff
@@ -7,10 +7,21 @@
       "root": "",
       "sourceRoot": "src",
       "projectType": "application",
+
+      "i18n": {
+        "sourceLocale": "en-US",
```

Comment:
Why "en-US"? It should rather be "en".

Reply:
Yes maybe it should, I took this from the documentation guide without thinking about it too much/